### PR TITLE
Remove the old docker-multinode in favor for the new one

### PR DESCRIPTION
**Pull request by luxas**
@johndmulhausen @orenye @mikedanese @pwittrock

### CODE REVIEW

File: docs/getting-started-guides/docker-multinode.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+Running Multi-Node Kubernetes Using Docker
```
Comment (johndmulhausen) [inline docs/getting-started-guides/docker-multinode.md]: The file must open with these lines:

---
---

Do not put the title -- that will be pulled in from guides.yml

File: docs/getting-started-guides/docker-multinode.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+v1.3.0 ships with support for amd64, arm and arm64. ppc64le isn't supported, due to a bug in the Go runtime, `hyperkube` (only!) isn't built for the stable v1.3.0 release, and therefore this guide can't run it. But you may still run Kubernetes on ppc64le via custom deployments.
+
+hyperkube was pushed for ppc64le at versions `v1.3.0-alpha.3` and `v1.3.0-alpha.4`, feel free to try them out, but there might be some unexpected bugs.
+
+### Options/configuration
+
+The scripts will output something like this when starting:
+
+```console
```
Comment (johndmulhausen) [inline docs/getting-started-guides/docker-multinode.md]: 45: ```shell

NOT console; must be a valid lexer from the list supported by Rouge

**Comment by luxas** [comment]
@johndmulhausen Updated

**Comment by luxas** [comment]
@mikedanese Ping

Also cc @kelseyhightower if you have time :)

File: docs/getting-started-guides/docker-multinode.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+
+## Prerequisites
+
+The only thing you need is a linux machine with **Docker 1.10.0 or higher**
+
+## Overview
+
+This guide will set up a 2-node Kubernetes cluster, consisting of a _master_ node which hosts the API server and orchestrates work
+and a _worker_ node which receives work from the master. You can repeat the process of adding worker nodes an arbitrary number of
```
Comment (mikedanese) [inline docs/getting-started-guides/docker-multinode.md]: For better or worse we have settled on using the terms "master" and "node". It's confusing to keep using worker and minion in documentation.

**Comment by mikedanese** [comment]
Please address comment in a follow up. @pwittrock this LGTM

**Comment by luxas** [comment]
Is it ok for me to merge this now?

I fixed @johndmulhausen's comments and @mikedanese LGTM'd it
I'll fix nits in follow-ups

**Pull request merged**